docs/content/reference/field.md

```diff
@@ -98,6 +98,22 @@ Example:
 sensitive: true
 ```
 
+### `write_only`
+If true, the field is considered "write-only", which means that its value will
+be obscured in Terraform output as we all as not be stored in state. This field is meant to replace `sensitive` as it doesn't store the value in state.
+See [Ephemerality in Resources - Use Write-only arguments](https://developer.hashicorp.com/terraform/language/resources/ephemeral/write-only)
+for more information.
+
+This field cannnot be used in conjuction with `immutable` or `sensitive`.
```

Comment:
```suggestion
This field cannot be used in conjuction with `immutable` or `sensitive`.
```

Reply:
Isn't `immutable` fine? The resource will recreate if you increment the version. Or does the _version_ need to be the one that's immutable?

Reply:
the version field (the non-write-only field) would need to be immutable.